src/components/20-molecules/list/pw.ui.test.js

```diff
@@ -0,0 +1,92 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-list';
+
+describe('List', () => {
+  it('should render', async () => {
```

Comment:
"should render" is a bit short, I would prefer "should render list" so its a bit more specific

Reply:
This was the default test. I wonder if I should just leave that one out... Isn't of much use anyways.

Reply:
- jest is printing out the group desc "List" and under that "should render". Thats ok.
- we need at least 1 test. A empty describe will throw an error.

Reply:
I leave the test, as we nowhere else check for visibility. Hope that's ok for you guys.

@danyball I agree, although I think something like "should render component" would be more describing and intuitive in my opinion.